CI-API.Application/ServiceInterface/IUserProfileService.cs

```diff
@@ -0,0 +1,23 @@
+﻿using CI_API.Core.ViewModel;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CI_API.Application.ServiceInterface
+{
+    public interface IUserProfileService
+    {
+        public Task<JsonResult> GetCountriesList();
```

Comment:
This should not be a part of UserProfileService and should be moved to other new Service

Reply:
Changes Done